platform/android/java/lib/src/org/godotengine/godot/GodotLib.java

```diff
@@ -219,6 +219,11 @@ public static native boolean initialize(Activity activity,
 	 */
 	public static native void requestPermissionResult(String p_permission, boolean p_result);
 
+	/**
+	 * Invoked on the theme change.
+	 */
+	public static native void themeChanged();
+
```

Comment:
For the Android implementation, let's be specific in the naming and description since this only reports changes from Dark to Light mode and vice-versa (e.g: can switch to `onNightModeChanged()`).